STYLEGUIDE.md

```diff
@@ -0,0 +1,101 @@
+- Where not otherwise contradicted by this guide or specific individual requirements, a subset of Markdown corresponding to that cross-compatible with reST should be preferred where possible; e.g. ``*italic*`` vs. ``_italic_``, ``*`` for bulleted lists, double backtick for inline code vs. single, etc.
+- Documentation should build with no warnings with ``Sphinx`` in nitpicky mode
+- Lower case filenames, rst extension, hyphens as space deliminators, short but descriptive
+- End files with a newline (blank line)
+- 1 blank line after all headings and before and after paragraphs, directives and "|"s
+- Use Line Feed (LF, ``\n``) as the End of Line character on all platforms
+- Use UTF-8 for all files (with no BOM, as dictated by the UTF-8 standard); do not use Windows Notepad as this adds a BOM and can otherwise corrupt files easily
+- Indicate special characters (e.g. em dash, en dash, etc) by simply including them inline with Unicode
+- Suggested to avoid non-ASCII characters where minimal downside/need exists, and recommended to keep used characters within the Latin-1 set where an additional range is not required by the specific usage, or an existing proper name or other identifier uses them
+- Spaces, no tabs for indentation and otherwise
+- Natural indents (3 spaces) for directives, 4 spaces for further indents beyond that in code
+- No hard wrap, put each sentence on its own line, and try to keep them under 180 characters
+    - Wrap code blocks at a width of 70 characters from the page margin for readability.
+- Place section links above the section with one blank line between them and the section, and two blank lines between sections (as appropriate for that section level) between the link and the preceding text.
+- Always use single backticks around link titles
+- Eliminate any trailing whitespace before commiting
+- Try to keep individual files to under around ~500 lines; split into separate topics beyond that
+
+
+
+## reST Style Elements
+
+- Filenames/paths: Use :file: and ``/`` for deliminators e.g. :file:`path/to/test.py`. Example filenames should be all lowercase with no spaces or special characters unless called for by the application; real filenames should follow their actual case. Use {descriptive-name} for placeholders.
+- Keyboard Shortcuts: Use :kbd: to format, Ctrl/Alt/Shift as abbreviations (instead of Command/Option), ``-`` with no spaces to separate, and title case (including capital letters for letter keys; always include Shift explicitly if needed) e.g. :kbd:`Ctrl-Alt-Shift-P`. Use the key name or symbol as printed on a standard keyboard. For any configurable shortcut, you should include "by default" to indicate this, and must include at least one another means the command can be accessed if available (e.g. via a menu or toolbar), which should be done for all shortcuts and should include all means of executing the command, if multiple others exist.
```

Comment:
another -> other ?

Reply:
```suggestion
- Keyboard Shortcuts: Use :kbd: to format, Ctrl/Alt/Shift as abbreviations (instead of Command/Option), ``-`` with no spaces to separate, and title case (including capital letters for letter keys; always include Shift explicitly if needed) e.g. :kbd:`Ctrl-Alt-Shift-P`. Use the key name or symbol as printed on a standard keyboard. For any configurable shortcut, you should include "by default" to indicate this, and must include at least one other means the command can be accessed if available (e.g. via a menu or toolbar), which should be done for all shortcuts and should include all means of executing the command, if multiple others exist.
```

Yup, it actually took me a minute to figure out that one myself, heh.